<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewNetworkWizardPage</class>
 <widget class="QWizardPage" name="NewNetworkWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <property name="subTitle">
   <string/>
  </property>
  <attribute name="pageId">
   <string notr="true"/>
  </attribute>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../qgama-core.qrc">:/images/gama-local-elipse.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Choose_Network_Type">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Choose Network Type:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Project">
        <property name="text">
         <string>&amp;Project:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_Project</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_Project">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_Network_Types">
        <property name="text">
         <string>&amp;Network Types:</string>
        </property>
        <property name="buddy">
         <cstring>listWidget_Network_Types</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QListWidget" name="listWidget_Network_Types">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_Description">
        <property name="text">
         <string>&amp;Description:</string>
        </property>
        <property name="buddy">
         <cstring>textBrowser_Description</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QTextBrowser" name="textBrowser_Description">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qgama-core.qrc"/>
 </resources>
 <connections/>
</ui>
